auth-web/src/components/auth/common/PaymentMethods.vue

```diff
@@ -140,6 +154,14 @@ const PAYMENT_METHODS = {
                   be requested when you are ready to complete a transaction.`,
     isSelected: false
   },
+  [PaymentTypes.EFT]: {
```

Comment:
There is already a call into payment_accounts, it should grab the `enable_eft` flag, if that's enabled it should provide this option to the client